example/Douban/test.hs

```diff
@@ -41,9 +39,10 @@ main = do
   print token
   case token of
     Right r -> do
+      -- TODO: display Chinese character. (Text UTF-8 encodeing does not work, why?)
       uid <- authGetBS mgr (accessToken r) [uri|https://api.douban.com/v2/user/~me|]
-      TL.putStrLn $ either TL.decodeUtf8 TL.decodeUtf8 uid
-    Left l -> BSL.putStrLn l
+      putStrLn $ either show (show . TL.decodeUtf8) uid
```

Comment:
It's need a few things in order to create keys there..
here is what's suppose to be. 
https://github.com/freizl/hoauth2/blob/572f70f9b014afeb8f0779ad2de28f3756e81a9d/example/Douban/test.hs

no worry, worst case is you'll break it and I can fix it later on. no big deal as long as it compiles.

https://github.com/freizl/hoauth2/blob/master/example/Douban/test.hs